app/modules/import-export-customization/assets/js/import/pages/import-complete.js

```diff
@@ -111,40 +116,46 @@ export default function ImportComplete() {
 							{ __( 'This website templates includes:', 'elementor' ) }
 						</Typography>
 					</Box>
-					<Stack spacing={ 2 } sx={ { pt: 1 } } >
-						<Box>
-							<Stack direction="row" alignItems="center" spacing={ 1 }>
-								<Typography variant="body2" color="text.primary" >
-									{ __( 'Site settings', 'elementor' ) }
+					<Stack spacing={ 2 } sx={ { pt: 1, maxWidth: '1075px' } } >
+						{ data.includes.includes( 'settings' ) && (
+							<Box>
+								<Stack direction="row" alignItems="center" spacing={ 1 }>
+									<Typography variant="body2" color="text.primary" >
+										{ __( 'Site settings', 'elementor' ) }
+									</Typography>
+									<ExternalLinkIcon />
+								</Stack>
+								<Typography variant="caption" color="text.secondary">
+									{ __( 'Global Colors | Global Fonts | Typography | Buttons | Images | Form Fields | Previousground | Layout | Lightbox | Page Transitions | Custom CSS', 'elementor' ) }
 								</Typography>
-								<ExternalLinkIcon />
-							</Stack>
-							<Typography variant="caption" color="text.secondary">
-								{ __( 'Global Colors | Global Fonts | Typography | Buttons | Images | Form Fields | Previousground | Layout | Lightbox | Page Transitions | Custom CSS', 'elementor' ) }
-							</Typography>
-						</Box>
-						<Box>
-							<Stack direction="row" alignItems="center" spacing={ 1 }>
-								<Typography variant="body2" color="text.primary" >
-									{ __( 'Content', 'elementor' ) }
+							</Box>
+						) }
+						{ data.includes.includes( 'content' ) && (
```

Comment:
@ManorHazaz @max-zu I think for each customization task going forward we can double check the summaries. I handled the plugins only.